views/lib/forms/form-auto-logout-alert.hbs

```diff
@@ -0,0 +1,48 @@
+<style>
+    .sloth {
+        width: 35%; 
+        margin-bottom: -50px;
+        height: auto; 
+        opacity: 0.6;
+    }
+
+    .sloth-text {
+        text-align: center;
+        padding: 15px;
+    }
+
+    @media (min-width: 575.98px){
+    .sloth {
+        float: right;
+        margin-top: -12px;
+        margin-bottom: -20px;
+        text-align: center;
+    }
+
+    .sloth-text {
+        text-align: left;
+        padding: 35px 0 10px 28px;
+    }
+    }
+</style>
+<div class="form-group" data-show-on-remaining-sec="{{#global "JWT_SHOW_TIMEOUT_WARNING_SECONDS"}}{{/global}}">
+<div class="text-center">
+    <img src="/images/sloth.svg" class="sloth sloth-default"/>
+    <img src="/images/sloth_error.svg" class="sloth sloth-error"/>
+    <div class="sloth-text sloth-default">
+        Achtung: Du wirst in <br>
+        <span style="color: red; font-size: larger;"><span class="js-time">n/a</span> Minuten</span> automatisch Ausgeloggt. <br>
+        Verlängere jetzt deine Sitzung auf zwei Stunden.
+    </div>
+    <div class="sloth-text sloth-error">
+        </br>
+        Uups… das hätte nicht passieren dürfen! </br> 
```

Comment:
hätte dürfen... klingt wie ein fehler von uns, sollte es nicht :)

Reply:
text has the ok of konsti and will be rewrite in future. In this case it is also our fault, because server is down or something like this.

Reply:
Es ist ein Fehler von uns, sollte also auch so klingen :)

Reply:
;(